src/Microsoft.ML.AutoML/Experiment/MetricsAgents/IMetricsAgent.cs

```diff
@@ -10,6 +10,6 @@ internal interface IMetricsAgent<T>
 
         bool IsModelPerfect(double score);
 
-        T EvaluateMetrics(IDataView data, string labelColumn);
+        T EvaluateMetrics(IDataView data, string labelColumn, string groupId);
```

Comment:
Is that mean that you need to pass null to `groupId` even for scenarios that don't require groupId?